dlp/docs/conf.py

```diff
@@ -164,7 +164,7 @@
 # Add any paths that contain custom static files (such as style sheets) here,
 # relative to this directory. They are copied after the builtin static files,
 # so a file named "default.css" will overwrite the builtin "default.css".
-html_static_path = ["_static"]
+# html_static_path = []
```

Comment:
Bogus

Reply:
This changes back the `docs/conf.py` which is not desired. Keep open until there's a new Artman version